samples/blog/css/algolia.css

```diff
@@ -1,3 +1,20 @@
+input[type="text"].algolia-magic-search-box--input,
+.algolia-magic-search-box--input {
```

Comment:
do we have elements with the same class .algolia-magic-search-box--input that are not input of type text?